src/pilot/assign_role.py

```diff
@@ -1366,13 +1366,21 @@ def change_physical_disk_state_wait(
     change_state_result = drac_client.change_physical_disk_state(
         mode, controllers_to_physical_disk_ids)
 
+    realtime_controllers = []
     job_ids = []
     if change_state_result['commit_required_ids']:
         for controller_id in change_state_result['commit_required_ids']:
+            realtime = drac_client.is_realtime_supported(controller_id)
+            realtime_controllers.append(realtime)
             job_id = drac_client.commit_pending_raid_changes(
-                controller_id, reboot=False, start_time=None)
+                controller_id, reboot=False, start_time=None,
+                realtime=realtime)
             job_ids.append(job_id)
 
+    # if all contollers support real time then is_reboot_required is false
+    if len(set(realtime_controllers)) == 1:
```

Comment:
this check looks good

Reply:
The above check will be true if either all of the controllers support realtime or all of the controllers don't support realtime.  I think we need to change this so that it also makes sure that the value is true.

Reply:
Updated in new PR[#254](https://github.com/dsp-jetpack/JetPack/pull/254)